### DISCUSSION TITLE
MySQL support

### DISCUSSION THREAD

**Discussion by rfriedr1** [biolab/orange3#6419] [Ideas]
Dear all,
I was wondering if there is a way to connect a MySQL Database. As I undertand, currently only Postgres or MSSQL is supported. Many many people use the free MySQL database, so I think this would be helpful.

Best
Ronny

**Comment by promexio** [comment]
Hi -
I do have same request!!
Nevertheless a big big thank to the developers, it's a beautiful piece of software.

---Gabriel

**Comment by simonaubertbd** [comment]
As discussed here https://github.com/biolab/orange3/issues/6435 an odbc support for reading and writing would be a game changer.

Best regards,

Simon
